docs/site/Discovering-models-from-relational-databases.md

```diff
@@ -0,0 +1,43 @@
+---
+lang: en
+title: "Discovering models from relational databases"
+keywords: LoopBack 4.0, LoopBack-Next
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Discovering-models-from-relational-databases.html
+---
+
+## Synopsis
+
+LoopBack makes it simple to create models from an existing relational database.
+This process is called _discovery_ and is supported by the following connectors:
+
+* [Cassandra](Cassandra-connector.html)
+* [MySQL](MySQL-connector.html)
+* [Oracle](Oracle-connector.html)
+* [PostgreSQL](PostgreSQL-connector.html)
+* [SQL Server](SQL-Server-connector.html)
+* [IBM DB2](DB2-connector.html)
+* [IBM DashDB](DashDB.html)
+* [IBM DB2 for z/OS](DB2-for-z-OS.html)
+* [SAP HANA](https://www.npmjs.org/package/loopback-connector-saphana) - Not officially supported; see [Community connectors](Community-connectors.html).
+
+
+## Overview
+Models can be discovered from a supported datasource by running the `lb4 discover` command.
+
+**The loopback project must be built and contain the built datasource files in `PROJECT_DIR/dist/src/datasources.js**
+
+### Options
+
+`--dataSource`: Put a valid datasource name here to skip the datasource prompt
+
+`--views`: Choose whether to discover views, default true
```

Comment:
`Choose whether to discover views, default true` -> `Choose whether to discover views.  Default is true`

Reply:
Done